### ISSUE TITLE
Update test_get_micro_data_get_calculator() for new TC

### ISSUE THREAD

**Issue by jdebacker** [PSLmodels/OG-USA#458]
Tax-Calculator has changed how reform policies are specified.  Thus we need to update the reform dictionary in `test_basic.test_get_micro_data_get_calculator()` to work with the latest version of TC.

We should also pin to a higher TC version in the `ogusa-dev` environment.

**Comment by rkasher** [comment]
Do you think that this is something I can handle on my own?

Best, Roy Kasher
________________________________

**Comment by jdebacker** [comment]
@rkasher You might be able to do it.  You'll want to edit this dictionary on lines 233-242 of `test_basic.py`:

```
    reform = {
        2017: {
            '_II_rt1': [.09],
            '_II_rt2': [.135],
            '_II_rt3': [.225],
            '_II_rt4': [.252],
            '_II_rt5': [.297],
            '_II_rt6': [.315],
            '_II_rt7': [0.3564],
        }, }
```

To be consistent with the changes in Tax-Calculator described [here](https://github.com/PSLmodels/Tax-Calculator/blob/master/UPGRADING.md#upgrading-to-tax-calculator-20).  Using the reform JSON files (like this [one](https://github.com/PSLmodels/Tax-Calculator/blob/master/taxcalc/reforms/2017_law.json) as examples of the changes in syntax might also be helpful.

**Comment by jdebacker** [comment]
Resolved with PR #460.
